shared-libraries/lib/utils/exchangeReq.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+Object.defineProperty(exports, "__esModule", {
+  value: true
+});
+exports.getExchangeRates = undefined;
+
+var _rates = require('shared-libraries/lib/fixtures/rates');
+
+var _rates2 = _interopRequireDefault(_rates);
```

Comment:
Again all these calls to babel's functions is kind of scary. Are we being forced to write such code?